lib/Entity/Widget.php

```diff
@@ -213,6 +213,12 @@ class Widget implements \JsonSerializable
      */
     public $isValid = false;
 
+    /**
+     * Flag to indicate whether the widget data has to be repeated in the player
+     * @var bool
+     */
+    public $isRepeatData = true;
```

Comment:
I think we are saving this in the widget options, so we shouldn't also need it here.